README.md

```diff
@@ -175,7 +178,7 @@ You can set these up in your env, so you always have them, or set them on the co
 #### Make targets
 
 - Use `make` to run `clippy`, build for the Pi using `cross`, build for the local machine using `cargo` and to run tests
-- Use `make pibuild` to build only for the Pi. This will build both `piggui` (with GUI and GPIO) and `piglet` binary
+- Use `make build` to build only for the Pi. This will build both `piggui` (with GUI and GPIO) and `piglet` binary
```

Comment:
"to build only for the pi" part is wrong and needs to be generalized. Maybe mention it detects if running on a Pi and does different things than if on a non-Pi host?